app.js

```diff
@@ -0,0 +1,111 @@
+const speakerContainer = document.querySelector('.speaker-container');
+
+const speakers = [
+  {
+    name: 'Itachi Uchiha',
+    title: 'The Extinguisher of the Uchiha Clan',
+    bio: 'Itachi Uchiha is a character in the Naruto manga and anime seriescreated by Masashi Kishimoto.',
+    image: './images/Itachi_Uchiha001.webp',
+  },
+  {
+    name: 'Naruto Uzumaki',
+    title: 'Hokage of the Hidden Leaf Village',
+    bio: 'The Current host of the Nine-Tails, Naruto Uzumaki is the main protagonist of the Naruto series.',
+    image: './images/naruto.png',
+  },
+  {
+    name: 'Sasuke Uchiha',
+    title: 'The Last Uchiha',
+    bio: 'Sasuke Uchiha is a fictional character in the Naruto manga and anime series created by Masashi Kishimoto.',
+    image: './images/sasuke.jpg',
+  },
+  {
+    name: 'Sakura Haruno',
+    title: 'The Medic Ninja',
+    bio: 'Sakura Haruno is a fictional character in the Naruto manga and anime series created by Masashi Kishimoto.',
+    image: './images/Sakura.jpg',
+  },
+  {
+    name: 'Kakashi Hatake',
+    title: 'The Copy Ninja',
+    bio: 'Kakashi Hatake is a fictional character in the Naruto manga and anime series created by Masashi Kishimoto.',
+    image: './images/kakashi.png',
+  },
+  {
+    name: 'Kisame Hoshigaki',
+    title: 'The Shark Ninja',
+    bio: 'Kisame Hoshigaki is a fictional character in the Naruto manga and anime series created by Masashi Kishimoto.',
+    image: './images/kisame.jpg',
+  },
+];
+
+function createSpeakers(speakers) {
+  for (let i = 0; i < speakers.length; i += 1) {
+    const speaker = speakers[i];
+
+    const speakerCard = document.createElement('div');
```

Comment:
I kindly suggest you use the semantic ARTICLE tag instead of DIV, this is the ideal tag for repetitive content.

Reply:
Done. Thank you for your reviews